blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -232,7 +232,7 @@ export const LeadArtPresentation = (props) => {
 								<div className={`${BLOCK_CLASS_NAME}__image-wrapper`}>
 									<Image
 										ansImage={galleryItem}
-										resizedOptions={{ smart: true }}
+										resizedOptions={{ smart: false }}
```

Comment:
I think this would work if you just remove this line... not passing in smart would not define it so it would not be true.

Reply:
```suggestion
```